src/visualCaptcha/Captcha.php

```diff
@@ -254,7 +276,7 @@ private function utilReadJSON( $filePath ) {
     }
 
     // Stream file from path
-    private function utilStreamFile( &$headers, $filePath ) {
+    private function utilStreamFile( $headers, $filePath ) {
```

Comment:
By removing the pointer here we no longer change the headers, so we'd have to return them, or they'd have to be set somewhere else, right? Or am I missing something?